webdevbroz/pomodoro-app
# Add pomodoro timer

## Goal
Create the Pomodoro Timer component and the integration tests for this component.

## How to test
1. Run `yarn test` making sure all tests pass. There will be React errors stating that the <svg> children elements such as <circle> or <g> can't be recognized by the browser. Ignore this for now as the tests still pass and the app works correctly, it's only an issue within the test simulated browser.
2. If all tests pass, check the timer visuals, comparing against the designs. Note: Responsiveness or changing timer settings has not been implemented yet.   3. Check timer functionality. Timer should countdown from 25:00 to 00:00 during focus time then 05:00 to 00:00, continuing in this loop. You will have to manually start each countdown ends.
Functionality to check:
- [ ] Focus time ends and switches to break time
- [ ] Break time ends and switches to focus time
- [ ] Pause the timer
- [ ] Start the timer

## Screenshots
![image](https://github.com/webdevbroz/pomodoro-app/assets/12942146/6acfdf64-5184-47ef-b581-f0ea5a19f35e)
![image](https://github.com/webdevbroz/pomodoro-app/assets/12942146/82c2ddf2-391b-4b83-9d46-9073682f55bb)
![image](https://github.com/webdevbroz/pomodoro-app/assets/12942146/6a680127-57f5-4ef8-825d-9b185fb4bf6e)
